### ISSUE TITLE
How to configure fonts path

### ISSUE THREAD

**Issue by omrqs** [Zevran/materialize-loader#31]
By default, path for fonts is "materialize-css/fonts/roboto/", but it's locate at "materialize-css/dist/fonts/roboto/".

How i can change this configuration? Can help me?

Error log:
```
react    | ERROR in ./~/css-loader!./~/sass-loader/lib/loader.js!./~/materialize-loader/materialize-styles.loader.js!./materialize.config.js
react    | Module not found: Error: Cannot resolve module 'materialize-css/fonts/roboto/Roboto-Medium.woff2' in /home/node/src
react    |  @ ./~/css-loader!./~/sass-loader/lib/loader.js!./~/materialize-loader/materialize-styles.loader.js!./materialize.config.js 6:110802-110861

```

**Comment by Cleomir** [comment]
I'm also having the same problem, I'm using materialize-css v0.99

**Comment by Zevran** [comment]
Hi @omrqs,

Thanks for reporting the error, I apologies for the fix delay.
It should be resolved with the latest version ```2.0.1``` 😄
